[2022-11-13T05:52:56.543+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_availible scheduled__2022-11-11T00:00:00+00:00 [queued]>
[2022-11-13T05:52:56.655+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_availible scheduled__2022-11-11T00:00:00+00:00 [queued]>
[2022-11-13T05:52:56.658+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-13T05:52:56.660+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-13T05:52:56.663+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-13T05:52:56.787+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_availible> on 2022-11-11 00:00:00+00:00
[2022-11-13T05:52:56.818+0000] {standard_task_runner.py:55} INFO - Started process 198 to run task
[2022-11-13T05:52:56.867+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_availible', 'scheduled__2022-11-11T00:00:00+00:00', '--job-id', '78', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmptn0ye40c']
[2022-11-13T05:52:56.881+0000] {standard_task_runner.py:83} INFO - Job 78: Subtask is_api_availible
[2022-11-13T05:52:57.660+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_availible scheduled__2022-11-11T00:00:00+00:00 [running]> on host bca668ad4dbc
[2022-11-13T05:52:58.486+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_availible
AIRFLOW_CTX_EXECUTION_DATE=2022-11-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-11T00:00:00+00:00
[2022-11-13T05:52:58.501+0000] {http.py:120} INFO - Poking: api/
[2022-11-13T05:52:58.650+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2022-11-13T05:52:58.701+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2022-11-13T05:52:59.460+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2022-11-13T05:52:59.524+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_availible, execution_date=20221111T000000, start_date=20221113T055256, end_date=20221113T055259
[2022-11-13T05:52:59.717+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-11-13T05:53:00.363+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
